backend/src/event/Event.ts

```diff
@@ -270,6 +380,34 @@ export class Event {
     return this
```

Comment:
it is not allowed to create a contribution without an amount, so please set it to mandatory

Reply:
can't see where you are meaning, I've solved it now in some places, but you may take a look